### PR TITLE
feat: make ticket holder info editable

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/7906

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/components/forms/orders/attendee-list.js [hunk 1]
```diff
@@ -25,25 +29,39 @@ export default class AttendeeList extends Component {
...
+  saveHolder(holder) {
+    holder.save();
+    this.data.save();
+  }
+
+  @action
+  async downloadTicketForAttendee(eventName, orderId, attendeeId) {
+    try {
+      if (!this.ticketDowloaded) {
+        await this.confirm.prompt(this.l10n.t('Please check the filled detail carefully. Once you Download ticket, the information can\'t be changed on it.'));
```
Comment (pc-beast) [inline app/components/forms/orders/attendee-list.js:41]: I didn't get this prompt while I downloaded the ticket. :thinking:  Other things worked though.

**Comment by maze-runnar** [comment]
Screenshot from 2021-10-22 08-07-29

**Comment by mariobehling** [comment]
This issue was mainly about the attendee form fields that are configured in the wizard. Can the attendee change what he/she entered here previously?

Screenshot from 2021-10-22 06-39-03

**Review by pc-beast — COMMENTED** [review]
image
Dropdown fields are not editable.

**Comment by mariobehling** [comment]
Was this closed accidentally? Ok, lets follow up elsewhere.

**Comment by maze-runnar** [comment]
Closed accidently.

**Comment by maze-runnar** [comment]
> image Dropdown fields are not editable.

can't make select editable really.

**Review by Rohit484 — APPROVED** [review]

**Review by Rohit484 — APPROVED** [review]

**Comment by mariobehling** [comment]
I have tested that in more detail now. Unfortunately the process is not yet ready for users. 

After an order is made the user can still change data, but there is no way to save it. It lacks a save button. Also changes are not reflected on the ticket. Expected would be a button "Edit Attendee Information" and "Save". Fields should only become editable after the user has pressed "Edit". This might have to implemented in a separate page. 

So, for now I am reverting this and I hope the remaining parts can be taken care of.

**Pull request merged**